src/modules/wallet/api/utils/getHwAccount.ts

```diff
@@ -0,0 +1,10 @@
+import { jotaiStore } from 'jotaiStore'
+
+import { hwAccountIndexAtom } from '../state'
+
+export function getHwAccount(): { index: number; path: string } {
+  const index = jotaiStore.get(hwAccountIndexAtom)
+  const path = `m/44'/60'/0'/0/${index}`
```

Comment:
I guess this is a good to have, but, shouldn't we allow users to pick a different derivation path?

Reply:
There is only one for Ethereum-like networks

Reply:
For this wallet?
Because I can use different paths - or at least different base indexes - with Ledger

Reply:
I don't know about the possibility to change the derivative path for Trezor.
https://docs.trezor.io/trezor-firmware/misc/coins-bip44-paths.html